velox/exec/AddressableNonNullValueList.h

```diff
@@ -57,6 +58,13 @@ class AddressableNonNullValueList {
       vector_size_t index,
       HashStringAllocator* allocator);
 
+  /// Append a non-null serialized value to the end of the list.
+  /// Returns (position, size) that can be used to access the
+  /// value later.
+  std::pair<HashStringAllocator::Position, size_t> appendSerialized(
```

Comment:
We do not need to return the size since it is the same as value.size(), no?

Reply:
In the past I thought that it was better to report size separately incase HashStringAllocator added some overhead bytes. But you are right that its not needed in the current implementation. Changing it.